packages/private/docker/package.json

```diff
@@ -3,10 +3,5 @@
   "version": "0.0.0",
   "private": true,
   "dependencies": {},
-  "devDependencies": {
-    "@types/ramda": "^0.27.0",
-    "@types/semver": "^7.0.0",
-    "ramda": "^0.27.0",
-    "semver": "^7.0.0"
-  }
+  "devDependencies": {}
```

Comment:
docker imports both `ramda` and `semver`

Reply:
I have moved them to the root `package.json`. Based on https://github.com/lerna/lerna#common-devdependencies I assume that collecting the devDependencies in one the repo is the way to go so that updating the dev dependencies is easier. See for example https://github.com/LittleBreak/lerna-best-practices -> there are no devDependencies in the child projects only on root level...

Reply:
Anyhow: I do not mind to move the devDependencies back to the projects using them

Reply:
Hm... Not a big fan tbh, since it makes it harder to see where dependencies are used. But your call :)